cpp/src/arrow/compute/kernels/aggregate_basic_internal.h

```diff
@@ -549,6 +549,28 @@ struct BooleanMinMaxImpl : public MinMaxImpl<BooleanType, SimdLevel> {
   }
 };
 
+template <SimdLevel::type SimdLevel>
+struct BooleanAnyImpl : public MinMaxImpl<BooleanType, SimdLevel> {
```

Comment:
I don't really understand why this is inheriting from MinMax. Does it help reduce the code size in any way?